components/compiler-families/gnu-compilers/SPECS/gnu-compilers.spec

```diff
@@ -24,52 +23,55 @@ Summary:   The GNU C Compiler and Support Files
 Name:      %{pname}%{PROJ_DELIM}
 Version:   %{gnu_version}
 Release:   1%{?dist}
-License:   GNU GPL
+License:   GPLv3 and GPLv3+ with exceptions and LGPLv3 and GPLv2 and LGPLv2+
 Group:     %{PROJ_NAME}/compiler-families
 URL:       http://gcc.gnu.org/
 Source0:   https://ftp.gnu.org/gnu/gcc/gcc-%{gnu_version}/gcc-%{gnu_version}.tar.xz
 Source1:   https://ftp.gnu.org/gnu/gmp/gmp-%{gmp_version}.tar.bz2
 Source2:   https://ftp.gnu.org/gnu/mpc/mpc-%{mpc_version}.tar.gz
 Source3:   https://ftp.gnu.org/gnu/mpfr/mpfr-%{mpfr_version}.tar.gz
 
-BuildRequires:  bison
-BuildRequires:  flex
-BuildRequires:  gettext-devel
-BuildRequires:  perl
+# Requirements from https://gcc.gnu.org/install/prerequisites.htmlzypper
 BuildRequires:  gcc-c++
-%if 0%{?suse_version} > 1220
-BuildRequires:  makeinfo
-%else
-BuildRequires:  texinfo
-%endif
-BuildRequires:  zlib-devel
-%if 0%{?sles_version} || 0%{?suse_version}
+BuildRequires:  binutils >= 2.30
+BuildRequires:  make >= 3.80
+BuildRequires:  gettext-devel >= 0.14.5
+BuildRequires:  flex >= 2.5.4
+BuildRequires:  texinfo >= 4.7
+BuildRequires:  m4 >= 1.4.6
+%if 0%{?sle_version} || 0%{?suse_version}
```

Comment:
Should line 79 below also be updated from `sles_version` to `sle_version` ?
Disclaimer: I don't know yet what is the difference between those but I noticed that it has changed here.

Reply:
Thanks, I've updated it to sle_version. sles_version is the older macro.